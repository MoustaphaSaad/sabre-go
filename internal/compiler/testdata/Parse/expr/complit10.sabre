[3]int{
	1: 42
}