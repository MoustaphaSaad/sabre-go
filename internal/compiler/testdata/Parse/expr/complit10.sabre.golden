>> 		1: 42
>> 		     ^
>> 	}
>> 	  
Error[internal/compiler/testdata/Parse/expr/complit10.sabre:2:7]: expected an expression but found ';(\n)'
